### ISSUE TITLE
Decorative image does not require the title attribute

### ISSUE THREAD

**Issue by ndolter** [mmiron-HC/WST-RSAMS#134]
URL: https://recalls-rappels.canada.ca/fr/avis-rappel/rappel-certains-produits-vegetaliens-et-vegetariens-marque-happy-veggie-world-hearts
User Agent: Chrome v98
Language: English & French

Issue Description: The fork and knife image are decorative and do not require a title attribute.  The current title attribute does not provide any meaning to screen reader users.  It adds no value considering the alt attribute is empty.  
	
Steps to reproduce:
1. Navigate to the url above. Or search on: Certain Happy Veggie World, Hearts Choices, and Vegefarm brand vegan and vegetarian products recalled due to undeclared egg
2. Navigate to the Related AWRs section
3. View the code or hover over the image with the mouse

Expected Result: To avoid any kind of confusion the title attribute should be removed from all decorative images.

![image](https://user-images.githubusercontent.com/68117281/159344062-479a12ca-97db-49a5-bdf5-e1328081376e.png)

**Comment by echouin** [comment]
@ndolter this is part of older migrated content that was exported as flat HTML. We made a decision at the outset of the project to import 19,000 older records which would inevitably have issues. In order to fix all of them, each of the 19,000 records would need to be re-entered into the new system by individual(s) which is simply not feasible at this time.

**Comment by mmiron-HC** [comment]
@echouin I understand what you are saying, however from our end, these are no resolved. We need these to stay open. We cannot close it until it is fixed. 
A suggestion would be to create another label such as "legacy" or something meaningful to differentiate these types of issues from the rest. I will let you decide.

**Comment by echouin** [comment]
Hi @mmiron-HC it will not be fixed for the 19,000 records. Yes, I concur with "Legacy". @davidgotlieb does that work for you?

**Comment by davidgotlieb** [comment]
Sure, Legacy works

**Comment by mmiron-HC** [comment]
@davidgotlieb @echouin tagged this one with legacy. Please tag the other ones that you feel fit this criteria.

**Comment by echouin** [comment]
@mmiron-HC could the Milestone be "legacy: not fixed"? I typically sort that way when auditing.

**Comment by mmiron-HC** [comment]
@echouin yes it can. I was too fast on the other one. Legacy w status of Not Fixed works.

**Comment by mmiron-HC** [comment]
@40999221 @zimw7 @YuxuanXie01  please retest and provide comment if fixed/fails, Thanks.

**Comment by YuxuanXie01** [comment]
Retested, not fixed. Decorative images still have title attribute. 
![image](https://github.com/user-attachments/assets/dedc4c8c-e9d9-40b8-95ba-0f5ebd73b877)

**Comment by zimw7** [comment]
Issue still exists. The title attribute has not been removed from the decorative images
![Snap6](https://github.com/user-attachments/assets/49174766-f2a7-4d41-8b76-5f60c8881d0e)

**Comment by 40999221** [comment]
Retested and issue still exist. Title attribute is still there on decorative images.

![image](https://github.com/user-attachments/assets/64a26986-9218-4c2e-a0e8-f23034b4ec67)

**Comment by mmiron-HC** [comment]
@zimw7  can you please bring this one over to Jira. When you have, please add a label of moved_to_Jira to this ticket.
